### ISSUE TITLE
duplicate games for different users

### ISSUE THREAD

**Issue by jon-eng** [jon-eng/videogame#5]
@harimohanraj89 @DrRobotmck @williamfshaw @bigtone1284

I have a many to many relationship between games and users.  I made a join table for them and also have a unique validation for the game title.  Right now, if a user saves a game he won't be able to save it again, however if I login to a different user, he can't save that same game. I want different users to be able to save the same game but not have a duplicate games under one user.  If I take out the unique validation from the game title, one user will be able to save the game multiple times under his profile, which I don't want.  I'm not too sure where to go from here to fix this issue.

**Comment by WilliamfShaw** [comment]
@jon-eng 

Please check slack for the proper template for your issue. Fill this out and resend.

**Comment by jon-eng** [comment]
CONTEXT

@harimohanraj89 @DrRobotmck @williamfshaw @bigtone1284

WHAT USER STORY YOU ARE WORKING ON:

I am working on the user story "As a user, I want to save a game without having duplicates."

I am working on the bug_fixes branch.

WHAT YOU ARE TRYING TO DO:

I want different users to be able to save the same game but not have duplicate games under one user.

DETAILED DESCRIPTION OF THE BUG/ERROR:

if a user saves a game he won't be able to save it again which is correct, however if I login to a different user, and also try to save that same game, it won't save.

WHAT I'VE TRIED

I have a many to many relationship between games and users. I made a join table for them and also have a unique validation for the game title. I've tried using findOrCreate but nothing will save, when I use that.

QUESTION

I don't know if I should change my "unique" validation or change my route to fix this

:heart: A meticulous and troubled student
